clouddriver-cloudfoundry/clouddriver-cloudfoundry.gradle

```diff
@@ -10,7 +14,9 @@ dependencies {
   compile spinnaker.dependency('lombok')
 
   spinnaker.group('retrofitDefault')
-  spinnaker.group('test')
+
+  testCompile('org.junit.jupiter:junit-jupiter-api:5.2.0')
```

Comment:
Is there a group similar to `spinnaker.group('test')` that we can create for this?

Reply:
Not that I know of, but a new one would be welcome: github.com/spinnaker/spinnaker-dependencies